demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1386,15 +1387,17 @@ export class DemoMeetingApp
       configuration.attendeePresenceTimeoutMs = Number(timeoutMs);
     }
     configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = this.enableSimulcast;
+    if (this.usePriorityBasedDownlinkPolicy) {
+      this.priorityBasedDownlinkPolicy = new VideoPriorityBasedPolicy(this.meetingLogger);
+      configuration.videoDownlinkBandwidthPolicy = this.priorityBasedDownlinkPolicy;
+      this.priorityBasedDownlinkPolicy.addObserver(this);
```

Comment:
Is this change related?

Reply:
> Is this change related?

Yes, this is to make sure we set the downlink policy in meetingSessionConfiguration before we use it to init the meetingSession, which is what we expect our customers to do.